From 73617ce2f39d6311231aff052cb790c692108f19 Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Fri, 21 Mar 2014 08:59:40 +0800
Subject: [PATCH 0332/1074] ENGR00304563-2 ARM: dts: add fixed phy address to
 imx6sx 17x17/19x19 arm2 boards

Add fixed phy-id property for imx6sx 17x17/19x19 arm2 boards.

Signed-off-by: Fugang Duan <B38611@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sx-17x17-arm2.dts |    2 ++
 arch/arm/boot/dts/imx6sx-19x19-arm2.dts |    2 ++
 2 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
index 185e269..f48bbf9 100644
--- a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
@@ -100,6 +100,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_enet1_1>;
 	phy-mode = "rgmii";
+	phy-id = <1>;
 	fsl,num_tx_queues=<3>;
 	fsl,num_rx_queues=<3>;
 	phy-supply = <&reg_rgmii_1v8>;
@@ -110,6 +111,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_enet2_1>;
 	phy-mode = "rgmii";
+	phy-id = <0>;
 	fsl,num_tx_queues=<3>;
 	fsl,num_rx_queues=<3>;
 	pinctrl-assert-gpios = <&max7322_1 0 GPIO_ACTIVE_HIGH>;
diff --git a/arch/arm/boot/dts/imx6sx-19x19-arm2.dts b/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
index 64805f9..2ef6900 100644
--- a/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-19x19-arm2.dts
@@ -63,6 +63,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_enet1_1>;
 	phy-mode = "rgmii";
+	phy-id = <1>;
 	fsl,num_tx_queues=<3>;
 	fsl,num_rx_queues=<3>;
 	pinctrl-assert-gpios = <&max7322_1 0 GPIO_ACTIVE_HIGH>;
@@ -73,6 +74,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_enet2_1>;
 	phy-mode = "rgmii";
+	phy-id = <0>;
 	fsl,num_tx_queues=<3>;
 	fsl,num_rx_queues=<3>;
 	pinctrl-assert-gpios = <&max7322_2 0 GPIO_ACTIVE_HIGH>;
-- 
1.7.5.4

